File: app/src/main/java/ru/demo/model/PokemonMaster.java
package ru.demo.model;

public class PokemonMaster {
    
    public PokemonMaster(String masterName, Pokemon pokemon) {
        System.out.println("Я покемон-мастер " + masterName + "!");
        System.out.println("Я тренирую его!");
        System.out.println(pokemon.pokemonImage());
        System.out.println("=======================");
    }
}

File: app/src/main/kotlin/ru/demo/model/PokemonMaster.kt
package ru.demo.model

class PokemonMaster(masterName: String, pokemon: Pokemon) {
    init {
        println("Я покемон-мастер $masterName!")
        println("Я тренирую его!")
        println(pokemon.pokemonImage)
        println("=======================")
    }
}
